debezium-core/src/main/java/io/debezium/transforms/ByLogicalTableRouter.java

```diff
@@ -24,20 +22,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import io.debezium.config.Configuration;
-import io.debezium.config.Field;
-import io.debezium.data.Envelope;
-import io.debezium.util.AvroValidator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
 
 /**
  * A logical table consists of one or more physical tables with the same schema. A common use case is sharding -- the
  * two physical tables `db_shard1.my_table` and `db_shard2.my_table` together form one logical table.
- *
+ * <p>
  * This Transformation allows us to change a record's topic name and send change events from multiple physical tables to
  * one topic. For instance, we might choose to send the two tables from the above example to the topic
  * `db_shard.my_table`. The config options {@link #TOPIC_REGEX} and {@link #TOPIC_REPLACEMENT} are used
  * to change the record's topic.
- *
+ * <p>
```

Comment:
On this one we should add the "<p>" as otherwise paragraphs won't be properly separated in rendered output, e.g. when hovering over a comment in Eclipse. +1 for handling it separately, though. I've filed https://issues.jboss.org/browse/DBZ-331 for adding this kind of information to CONTRIBUTING.md.